release-notes/README.md

```diff
@@ -4,6 +4,16 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## May 9, 2022
+
+### iOS AEPCore 3.6.0
+
+* App Extension Support
+* Add a new API to the Extension protocol for getting latest non pending shared state
```

Comment:
"Added a new API to the Extension protocol for getting the latest non-pending shared states".